test/renate/updaters/experimental/test_fine_tuning.py

```diff
@@ -20,13 +20,23 @@ def get_model_and_dataset():
     return model, dataset
 
 
-def test_fine_tuning_updater():
+@pytest.mark.parametrize("devices", [None, 1])
+@pytest.mark.parametrize("strategy", [None, "ddp"])
+@pytest.mark.parametrize("accelerator", ["cpu"])
+def test_fine_tuning_updater(devices, strategy, accelerator):
```

Comment:
aren't all tests executing the same logic?

Reply:
This is only a sanity check to ensure that  all(-most) possible inputs work.